src/common.rs

```diff
@@ -304,7 +295,7 @@ impl<Ext> CSD<Ext> {
 /// Error and state information of an executed command
 ///
 /// Ref PLSS_v7_10 Section 4.10.1
-#[derive(Debug, Clone, Copy)]
+#[derive(Clone, Copy)]
```

Comment:
I think this was a mistake, right?

Reply:
Surprisingly this is actually intended.

For the `SD` implementation, deriving Debug here doesn't do anything since [`SD` does not](https://github.com/jkristell/sdio-host/blob/master/src/sd.rs#L6) implement Debug and hence `CardStatus<SD>` doesn't either. Instead we have a manual implementation in `src/sd.rs`

`CardStatus<EMMC>` would derive a Debug implementation as `EMMC` implements Debug. Since I added a [manual implementation](https://github.com/jkristell/sdio-host/pull/14/files#diff-d10776a80f750e53e770f74a30451a16071eb67e5e60f792da983b3f96c05878R155) of Debug in `src/emmc.rs`, I needed to remove the conflicting derived implementation.